### DISCUSSION TITLE
Why do I get this npm ERR! cb.apply is not a function

### DISCUSSION THREAD

**Discussion by cruz-emman** [vercel/next.js#48853] [Help]
### Summary

so I'm trying to learn a Next.js and whenever I use npx create-next-app@latest, I get this error

```
npm ERR! cb.apply is not a function

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Local\npm-cache\_logs\2023-04-26T06_03_39_459Z-debug.log

Aborting installation.
  npm install --save-exact --save react react-dom next typescript @types/react @types/node @types/react-dom tailwindcss postcss autoprefixer eslint eslint-config-next has failed.
```

 tried doing these.

npm cache clear --force and this is what I get

npm WARN using --force Recommended protections disabled.

I also deleted the` npm folder` in `AppData/Roaming,` and didn't work.

I uninstall the `Node` and reinstalled it again.

My npm version and node version right now is

`npm 9.5.1`

`node v18.16.0`

![Uploading Screenshot 2023-04-26 154610.png…]()

**Comment by ciruz** [comment]
I had some `node.js` issues myself in the last couple of weeks, on Windows. So I installed [`nvm`](https://github.com/nvm-sh/nvm) and uninstalled `node.js` afterwards. I now manage my `node.js`  installations with `nvm`, no problems anymore.

Bonus: I also use `yarn` on Windows, not `npm` (`npx`).
